gradle.properties

```diff
@@ -106,8 +106,6 @@ artifactRedirecting.androidx.compose.version=1.8.0-alpha03
 artifactRedirecting.androidx.compose.material3.version=1.4.0-alpha01
 artifactRedirecting.androidx.compose.foundation.version=1.8.0-alpha03
 artifactRedirecting.androidx.compose.material.version=1.8.0-alpha03
-# The latest version is not published yet: https://mvnrepository.com/artifact/androidx.compose.material/material-navigation
-artifactRedirecting.androidx.compose.material.material-navigation.version=1.7.0-beta01
```

Comment:
This setting should point to `1.8.0-alpha03` now

Reply:
It uses `artifactRedirecting.androidx.compose.material.version=1.8.0-alpha03`

Reply:
https://github.com/JetBrains/compose-multiplatform-core/blob/b42476d5512b2a5bc94c1e5c75b0526e0eeff0ef/buildSrc/public/src/main/kotlin/androidx/build/jetbrains/ArtifactRedirecting.kt#L43

Reply:
Ah! yep, I see